packages/serialize/src/index.js

```diff
@@ -264,7 +264,12 @@ function createStringFromObject(
 
   if (Array.isArray(obj)) {
     for (let i = 0; i < obj.length; i++) {
-      string += handleInterpolation(mergedProps, registered, obj[i], false)
+      string += `${handleInterpolation(
```

Comment:
@mitchellhamilton any ideas when this could break stuff? I cant find any, so I think this is safe to do, but would be great to give this a little bit more extra thought.